zerver/data_import/rocketchat.py

```diff
@@ -0,0 +1,679 @@
+import logging
```

Comment:
Is there top-level documentation on the Rocket.Chat data export format that we could link to from a comment located here?

Reply:
(Might be worth adding something similar for the other data import tools, if we haven't already).

Reply:
There isn't any such documentation for Rocket.Chat (they don't even have any mechanism to export data, we use the mongo dump in our import tool). But it would be worth adding this in other import tools.